### DISCUSSION TITLE
mod_auth_oidc - Logout  missing id_token_hint...need help

### DISCUSSION THREAD

**Discussion by kbejjipuram** [OpenIDC/mod_auth_openidc#975] [Q&A]
We use below configuration to make sure the AuthN/AuthZ works for multiple Apache web server instances under load balancer. All works well but after adding this we are noticing the logout redirect is NOT adding id_token_hint anymore which is resulting in inconsistent behavior upon logout.

OIDCSessionType client-cookie

Could you please check and suggest if we need to add any additional OIDC property to Apache to get this id_token_hint to be added to logout URL.

Upon logout we just see this post_logout_redirect_uri is being added and we lost this id_token_hint !!!

**Comment by zandbelt** [comment]
see https://github.com/zmartzone/mod_auth_openidc/discussions/812, you'll have to use version >= 2.4.12 and set `OIDCSessionType client-cookie:store_id_token` as documented in https://github.com/zmartzone/mod_auth_openidc/blob/v2.4.12.1/auth_openidc.conf#L571-L572
